README.md

```diff
@@ -62,6 +62,80 @@ an easy way to setup bitbake based projects. For more details, visit the
 
 For a manual build without KAS, refer to the [Yocto Project Quick Build](https://docs.yoctoproject.org/brief-yoctoprojectqs/index.html).
 
+## Flash
+
+### Build QDL tool
+
+QDL tool communicates with USB devices of id `05c6:9008` and uploads a flash
```

Comment:
change "id" to "VID:PID"? qdl can work with multiple devices connected to the same host using the device unique ID. This is done by adding flag "--serial"

Reply:
That's from QDL tool readme https://github.com/linux-msm/qdl/blob/master/README, based on that I assumed that all Q devices use the same `VID:PID` in EDL mode

Reply:
That's correct, but if you have multiple devices connected to the same host, they will have different serial ID: https://github.com/linux-msm/qdl/blob/master/qdl.c#L131

Reply:
ah, ok, fill fix

Reply:
fixed